src/main/java/mekanism/common/tile/TileEntityBoundingBlock.java

```diff
@@ -41,10 +51,18 @@ public void validate() {
         }
     }
 
+    public TileEntity getMainTile() {
+        if (receivedCoords && world.isBlockLoaded(getMainPos())) {
```

Comment:
This doesn't look like it will currently work on the client, as that field is not set there (given the setter for `mainPos` isn't getting used)

Reply:
I didn't see any issues in testing but its hard to tell with the amount of legacy code left laying around. I'll take a look first chance I get but will be working 10h days this week so might not get to it right away.